apps/transport/lib/transport_web/templates/backoffice/contact/_notification_subscriptions.html.heex

```diff
@@ -51,10 +51,10 @@
     <%= submit("Mettre à jour les abonnements") %>
   <% end %>
 
-  <h3>Abonnements existants</h3>
+  <h3><%= dgettext("backoffice", "Existing subscriptions") %></h3>
 
   <p :if={Enum.empty?(@notification_subscriptions)} class="notification">
-    Il n'y a pas d'abonnements à des notifications pour ce contact.
+    <%= dgettext("backoffice", "There's no notification subscriptions for this contact.") %>
```

Comment:
There are no notification subscriptions il me semble

https://twominenglish.com/there-is-no-or-there-are-no/